CRM/Report/Form.php

```diff
@@ -714,7 +705,7 @@ public function preProcess() {
         }
       }
 
-      $doNotCopy = array('required');
+      $doNotCopy = array('required', 'default');
```

Comment:
this is the change - ie. if the DAO defaults to 1 then that doesn't mean the field should default to displayed or being a group by